### PR TITLE
Workflow update

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -94,3 +94,48 @@ jobs:
...
+    name: CodeQL
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more:
+        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
```
Comment (AnnaShaleva) [inline .github/workflows/go.yml:108]: Do we really need all these CodeQL-related comments in every repo? If we miss something, then we can go to CodeQL action documentation and take a look there.

**Review by AnnaShaleva — COMMENTED** [review]
Otherwise LGTM.

File: .github/workflows/go.yml (hunk 1 above)
Comment (roman-khimov) [reply .github/workflows/go.yml:108]: That's the way it adds an action by default. Copy/pasting is much easier than thinking.

**Review by AnnaShaleva — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]
> workflows: make test workflow actually do something!
> One of the best things I've ever seen: a test workflow that does nothing.

let's write the text more formal:
* w/o exclamation marks
* w/o personalization

**Review by smallhive — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**